includes/class-newspack-popups.php

```diff
@@ -640,6 +641,23 @@ public static function archive_campaign( $id, $status ) {
 		update_term_meta( $id, self::NEWSPACK_POPUPS_TAXONOMY_STATUS, $status ? 'archive' : '' );
 	}
 
+	/**
+	 * Prevent setting the default category when publishing.
+	 *
+	 * @param string $new_status New status.
+	 * @param string $old_status Old status.
+	 * @param bool   $post Post.
+	 */
+	public static function remove_default_category( $new_status, $old_status, $post ) {
+		if ( self::NEWSPACK_POPUPS_CPT === $post->post_type && 'publish' === $new_status ) {
+			$default_category_id = (int) get_option( 'default_category', false );
+			$popup_has_category  = has_category( $default_category_id, $post->ID );
```

Comment:
It's hard to ascertain the editor's intention in the use case where they may want to intentionally apply the default category. But maybe we can be a bit less heavy-handed by checking not just if the post has the default category, but if the default category is the post's _only_ category term?